drivers/cc110x/cc110x-netdev2.c

```diff
@@ -84,6 +84,8 @@ static inline int _get_iid(netdev2_t *netdev, eui64_t *value, size_t max_len)
     cpuid_get(cpuid);
 
     memcpy(eui64 + 8 - n, cpuid, n);
+    eui64[sizeof(eui64_t) - 2] = 0;
+    eui64[sizeof(eui64_t) - 1] = cpuid[CPUID_LEN-1];
```

Comment:
Fine if it works for you... but I would argue to also set `radio_address` to this value. I don't have a board with cc11xx. So I reassign.